tests/keras/preprocessing/image_test.py

```diff
@@ -55,17 +55,18 @@ def test_image_data_generator(self, tmpdir):
                 vertical_flip=True)
             generator.fit(images, augment=True)
 
-            for x, y in generator.flow(images, np.arange(images.shape[0]),
+            N = images.shape[0]
```

Comment:
Avoid capitalized variable names and single-letter variable names; `batch_size` or `num_samples` would be better here.